client/Input.zig

```diff
@@ -66,6 +84,31 @@ pub fn handleKeyEvent(self: *Input, event: sdl3.events.Keyboard) void {
     }
 }
 
+pub fn handleMouseButtonEvent(self: *Input, event: sdl3.events.MouseButton) void {
+    const button_state = switch (event.button) {
+        .left => &self.left_click_held,
+        .middle => &self.middle_click_held,
+        .right => &self.right_click_held,
+        .x1 => &self.x1_click_held,
+        .x2 => &self.x2_click_held,
+        _ => null, // Resonite doesn't handle further buttons.
+    };
+
+    if (button_state == null)
+        return;
+
+    button_state.?.* = event.down;
+}
+
+pub fn handleMouseMotionEvent(self: *Input, event: sdl3.events.MouseMotion) void {
+    // Y is inverted for some reason???
+    const delta: math.Vector2f = .{ .x = event.x_rel, .y = -event.y_rel };
+    self.mouse_delta = self.mouse_delta.add(delta);
+
+    self.mouse_window_pos = .{ .x = event.x, .y = event.y };
+    self.mouse_desktop_pos = self.mouse_window_pos;
```

Comment:
Let's open an issue for this to implement it.

Reply:
#81